src/agent/samples/policy/set-policy-allowed/set-policy-allowed-data.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file will not be used anymore when/if we'll adopt the rootfs.sh changes for Policy + OPA.